OtripleS.Web.Api.Tests.Unit/Services/CourseAttachments/CourseAttachmentServiceTests.Exceptions.RetrieveById.cs

```diff
@@ -0,0 +1,169 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using OtripleS.Web.Api.Models.CourseAttachments;
+using OtripleS.Web.Api.Models.CourseAttachments.Exceptions;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace OtripleS.Web.Api.Tests.Unit.Services.CourseAttachments
+{
+    public partial class CourseAttachmentServiceTests
+    {
+        [Fact]
+        public async Task ShouldThrowDependencyExceptionOnRetrieveWhenSqlExceptionOccursAndLogItAsync()
+        {
+            // given
+            Guid someAttachmentId = Guid.NewGuid();
+            Guid someCourseId = Guid.NewGuid();
+            SqlException sqlException = GetSqlException();
+
+            var expectedCourseAttachmentDependencyException = 
+                new CourseAttachmentDependencyException(sqlException);
+
+            this.storageBrokerMock.Setup(broker =>
+                 broker.SelectCourseAttachmentByIdAsync(someCourseId, someAttachmentId))
+                    .ThrowsAsync(sqlException);
+
+            // when
+            ValueTask<CourseAttachment> retrieveCourseAttachmentTask =
+                this.courseAttachmentService.RetrieveCourseAttachmentByIdAsync
+                    (someCourseId, someAttachmentId);
```

Comment:
Every variable should have its own line if it exceeds the same line where the method is called as it clearly says here in the coding standard 

https://github.com/hassanhabib/CSharpCodingStandard/blob/master/Methods.md#dont-11

Reply:
Do the same for the rest